### ISSUE TITLE
d0 bias of displaced tracking

### ISSUE THREAD

**Issue by tomalin** [cms-L1TK/cmssw#138]
The fitted d0 of tracks from the displaced tracking is biased with respect to the truth. The PR https://github.com/cms-L1TK/cmssw/pull/124 eliminated some of the bias, but as the plots ( from @cgsavard )that can be found inside the review of that PR by show, some bias remains. This is not understood.

**Comment by skinnari** [comment]
@cgsavard you mentioned in #124 that you thought another correction is needed, correlated with the z shift. can you please elaborate? is this based on looking at the displaced math from @gershtein ?

**Comment by cgsavard** [comment]
Yes, we made a correction to d0 which fixed one of the biases seen but not the other. Therefore, I thought another higher order may be needed from the displaced math Yuri calculated. I spent some trying trying out a few orders to see if I could fix it but did not manage to do so. It's possible that some other fix is needed, not corresponding to the displaced math, but I'm not certain.
